style.css

```diff
@@ -0,0 +1,241 @@
+}
+
+/* heading style */
+#heading {
+    border-bottom: solid 1px grey;
+    background-color: lightgrey;
+    height: 20em;
+    display: grid;
+    grid-template: 1fr 1fr .5fr 1fr / 1fr 1fr 1fr;
+    /* flex-direction: column;
+    align-items: center; */
+    /* justify-content: center; */
+}
+#heading h1 {
+    /* border: solid; */
+    grid-column-start: 2;
+    grid-row-start: 2;
+    text-align: center;
+    font-weight: bold;
+}
+/* TODO: why is text not centered in border box */
+#heading sub {
+    /* border: solid; */
+    margin-bottom: 15px;
+    grid-column-start: 2;
+    grid-row-start: 3;
+    text-align: center;
+    font-weight: bold;
+    font-size: 1.9vw;
+    overflow-wrap: break-word;
+    position: static; 
+    line-height: normal;
+}
+#heading a {
+    border: solid;
+    margin: auto;
+    border-radius: 5px;
+    padding: 10px;
+    /* height: 50%; */
+    grid-column-start: 2;
+    grid-row-start: 4;
+    text-align: center;
+    /* font-size: 1px; */
+    color: grey;
+    text-decoration: none;
+}
+#heading a:hover {
+    color: yellow;
+}
+
+/* QUOTE styling (2) */
+.quote {
+    border-bottom: solid 1px grey;
+    background-color: antiquewhite;
+    height: 15em;
+    display: grid;
+    grid-template: 1fr 1fr 1fr 1fr / 1fr 5fr 1.5fr .5fr;
+    font-style: italic;
+}
+.quote blockquote {
+    /* border: solid; */
+    line-height: 200%;
+    grid-column-start: 2;
+    margin-left: 20px;
+    margin-right: 20px;
+    grid-row-start: 2;
+}
+.quote p::before {
+    content: "--";
+}
+.quote p {
+    /* border: solid; */
+    grid-column-start: 3;
+    grid-row-start: 3;
+}
+
+/* WHY styling */
+#why {
+    /* background-color: lavender; */
+    height: 20em;
+    border-bottom: solid 1px;
+}
+#why h2, #team h2, #sponsors h2{
+    /* border: solid; */
+    text-align: center;
+    width: 100%;
+    padding-top: 3%;
+    padding-bottom: 3%;
+}
+#why ul {
+    /* border: solid; */
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    padding-left: 5%;
+}
+
+#why li {
+    /* border: solid; */
+    width: 380px;
+    display: grid;
+    grid-template: .5fr 2fr / 40% 60%;
+}
+#why img { 
+    /* border: solid 1px; */
+    grid-row: auto / span 2;
+    height: 8em;
+}
+#why strong {
+    /* border: solid; */
+}
+#why p {
+    /* border: solid; */
+    /* word-wrap: none; */
+   width: 220px;
+} 
+
+/* TEAM styling */
+#team {
+    /* background-color: darkgreen; */
+    width: 100%;
+}
+/* container for team members */
+#team ul{
+    /* border: solid; */
+    list-style: none;
+    width: 700px;
+    margin: auto;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: space-evenly;
+}
+#team li{
+    /* border: solid; */
+    width: 30%;
+    display: grid;
+    grid-template-rows: 3fr .5fr .5fr;
+}
+#team li img {
+    border-radius: 80px;
+    margin: auto;
+}
+#team li span{
+    /* border: solid; */
+    text-align: center;
```

Comment:
It's really cool how you implemented grid inside the flex box! I learned something new :)